atlp-rwanda/c4-barefoot-frontend
# Ft sending travel request

# Description

This set up a project from scratch without using npx create-react-app


## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- Clone this repo
- Run the command `npm i`
- Run the command `npm start`
- Head over to this endpoint http://localhost:5000/requester/create-travel-request and make sure you are logged in and you will be able to create the trave request and run the tests to see the tested files
# Checklist:


# Screenshots (if applicable)
